health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -147,9 +147,9 @@ const addNewAssumption = () => {
                           )
                     })}
 
-                <div style={{background:"#eee", height:"0.2rem", marginBottom:"1.5rem"}}></div>
+                <div style={{background:"#eee", height:"0.2rem",margin:"1.5rem 1.0rem"}}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)